examples/Atlas/+atlasParams/Recovery.m

```diff
@@ -0,0 +1,21 @@
+classdef Recovery < atlasParams.Walking
+  methods
+    function obj = Recovery(r)
+      obj = obj@atlasParams.Walking(r);
+      obj.body_motion(r.foot_body_id.right).Kp = 10 * [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.left).Kp = 10 * [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.right).weight = 1e-3;
+      obj.body_motion(r.foot_body_id.left).weight = 1e-3;
+      obj.body_motion(r.findLinkId('pelvis')).weight = 1e-3;
+
```

Comment:
Where are these weights coming from? I don't think they match what we're using the control the robot.